docs/ops/comparison/Equal_1.md

```diff
@@ -23,7 +23,7 @@ o_{i} = a_{i} == b_{i}
   * **Range of values**:
     * *none* - no auto-broadcasting is allowed, all input shapes should match,
     * *numpy* - numpy broadcasting rules, description is available in [Broadcast Rules For Elementwise Operations](../broadcast_rules.md),
-    * *pdpd* - PaddlePaddle-style implicit broadcasting, description is available in [Broadcast Rules For Elementwise Operations](../broadcast_rules.md).
+    * *paddle* - PaddlePaddle-style implicit broadcasting, description is available in [Broadcast Rules For Elementwise Operations](../broadcast_rules.md).
```

Comment:
really? I suppose it's an artifacts of global replacement

Reply:
reverted